low-level/framework/rabbitmq/rabbitmq_ingress_processor.py

```diff
@@ -231,59 +230,26 @@ def _process_msg(self, ch, method, properties, body):
-                                                                 'localhost')
-
-            # Read RabbitMQ configuration for sensor messages
-            self._queue_name    = Conf.get(SSPL_CONF, f"{self.RABBITMQPROCESSOR}>{self.QUEUE_NAME}",
-                                                                 'actuator-req-queue')
-            self._exchange_name = Conf.get(SSPL_CONF, f"{self.RABBITMQPROCESSOR}>{self.EXCHANGE_NAME}",
-                                                                 'sspl-in')
-            self._routing_key   = Conf.get(SSPL_CONF, f"{self.RABBITMQPROCESSOR}>{self.ROUTING_KEY}",
-                                                                 'actuator-req-key')
-            self._username = Conf.get(SSPL_CONF, f"{self.RABBITMQPROCESSOR}>{self.USER_NAME}",
-                                                                 'sspluser')
-            self._password = Conf.get(SSPL_CONF, f"{self.RABBITMQPROCESSOR}>{self.PASSWORD}",'')
-
-            cluster_id = Conf.get(GLOBAL_CONF, f"{CLUSTER}>{self.CLUSTER_ID_KEY}",'CC01')
-
-            node_id = Conf.get(GLOBAL_CONF, f"{CLUSTER}>{SRVNODE}>{self.NODE_ID_KEY}",'SN01')
-            # Decrypt RabbitMQ Password
-            decryption_key = encryptor.gen_key(cluster_id, ServiceTypes.RABBITMQ.value)
-            self._password = encryptor.decrypt(decryption_key, self._password.encode('ascii'), "RabbitMQingressProcessor")
-
-            # Create a routing key unique to this instance
-            unique_routing_key = f'{self._routing_key}_node{node_id}'
-            logger.info(f"Connecting using routing key: {unique_routing_key}")
-            self._connection = RabbitMQSafeConnection(
-                self._username, self._password, self._virtual_host,
-                self._exchange_name, unique_routing_key, self._queue_name
-            )
-        except Exception as ex:
-            logger.error("RabbitMQingressProcessor, _configure_exchange: %r" % ex)
+        cluster_id = Conf.get(GLOBAL_CONF, f"{CLUSTER}>{self.CLUSTER_ID_KEY}",'CC01')
+        self._node_id = Conf.get(GLOBAL_CONF, f"{CLUSTER}>{SRVNODE}>{self.NODE_ID_KEY}",'SN01')
+        self._consumer_id = Conf.get(GLOBAL_CONF, f"{self.RABBITMQPROCESSOR}>{self.CONSUMER_ID}",
+                                            'sspl_actuator')
+        self._consumer_group = Conf.get(GLOBAL_CONF, f"{self.RABBITMQPROCESSOR}>{self.CONSUMER_GROUP}",
+                                            'cortx_monitor')
+        self._message_type = Conf.get(GLOBAL_CONF, f"{self.RABBITMQPROCESSOR}>{self.MESSAGE_TYPE}",
+                                            'Requests')
+        self._offset = Conf.get(GLOBAL_CONF, f"{self.RABBITMQPROCESSOR}>{self.OFFSET}",
+                                            'earliest')
+
 
     def shutdown(self):
         """Clean up scheduler queue and gracefully shutdown thread"""
         super(RabbitMQingressProcessor, self).shutdown()
-        try:
```

Comment:
there is no connection close in message bus? If not, check in kafka client lib & if exists, we should ask message lib to provide that.

Reply:
`there is no connection close in message bus?` As of now there is no way to close connection in message bus.
`check in kafka client lib & if exists, we should ask message lib to provide that` Yes, I will check this and report to foundation team